### DISCUSSION TITLE
I'm making a 100% automated fork of this, and I'm not sure how to handle images.

### DISCUSSION THREAD

**Discussion by aProfessionalFrog** [noahgsolomon/brainrot.js#15] [Q&A] [answered]
I've been working on a fork of brainrot.js called [autorot.js](https://github.com/aProfessionalFrog/autorot.js) that will automatically post the generated videos to your choice of brainrotted platform. It will probably run in a Docker container, just for ease of setup and running it. I have already minimized API keys by converting elevenlabs tts to a python package called piper-tts, that runs quickly on CPU, and getting rid of all OpenAI integrations in favor of Groq. 
Here's the problem: I got rid of Dalle-3 because the rate limit was causing issues. I could use the existing Google image search API, but as said in the readme, it is a pain to set up, and the purpose of getting rid of APIs was to make it easier to use. Another though I had was to run an image generator locally, but that would be both slow and take up a lot of storage. My final idea was to preload a folder with a few hundred images, and use some sort of text classification to pick the most suitable one for the current line of text.
If anyone has any feedback or ideas for what I should do about images, please let me know.

**Comment by noahgsolomon** [comment, marked as answer]
The simplest idea which would still make the video convincing would be to use different images of each agent as the top 1/2 image portion. And in this way it would be very easy to get an image. You could have images categorized my agent and emotion. So for each dialogue, you could have a model output an emotion of n types (e.g. HAPPY, SAD, ANGRY, ANNOYED, CONFUSED, etc.) and then name the image files as ${AGENT_NAME}-${EMOTION}.png
